lib/compat/wordpress-6.5/interactivity-api/interactivity-api.php

```diff
@@ -178,3 +178,23 @@ function wp_interactivity_data_wp_context( array $context, string $store_namespa
 		'\'';
 	}
 }
+
+if ( ! function_exists( 'data_wp_context' ) ) {
+	/**
+	 * `data_wp_context()` was renamed to follow WordPress Core naming schemes.
+	 *
+	 * @link https://github.com/WordPress/gutenberg/pull/59465/
+	 * @link https://core.trac.wordpress.org/ticket/60575
+	 *
+	 * @since 6.5.0
+	 *
```

Comment:
Add @deprecated 6.5.0

Reply:
Done in https://github.com/WordPress/gutenberg/pull/59834/commits/169a2e0959372c8c389d792a6101642abd69dc87